pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,113 @@
 
 <%= toc %>
 
-A *command* step runs a command on an agent. The following is the most simple *command* step:
+
+A *command* step runs a command on an agent. The command can be the name of a script, or <cli things>
```

Comment:
What about something like?

> A *command* step runs one or more shell commands on an agent. Each command can be a shell command, such as `npm install`, or an executable file or script, such as `build.sh`

Unfortunately you can't use a normal YAML "list" (like we support for a lot of other things):

```yaml
steps:
  - name: ":hammer:"
    command:
      - "build.sh"
      - "test.sh"
```

Instead you have to do one of these:

```yaml
steps:
  - name: ":hammer:"
    command: "build.sh && test.sh"
  - name: ":hammer:"
    command: "build.sh\ntest.sh"
```

But that's a bit sucky! Let me see if I can do a sneaky pull request this week to support the proper YAML list syntax?

Should we also support `commands` as a pseudonym as well?

```yaml
steps:
  - name: ":hammer:"
    commands:
      - "build.sh"
      - "test.sh"
```